assets/app/styles/_tile.less

```diff
@@ -14,6 +14,11 @@
   h1, h2, h3 {
     margin: (@line-height-computed / 2) 0px;
   }
+  .tile-details {
```

Comment:
did you check everywhere we use tile-details to make sure it doesn't cause any regressions?